src/applications/caregivers/definitions/caregiverUI.js

```diff
@@ -40,41 +37,51 @@ export default {
       'ui:title': 'Email Address',
       'ui:widget': 'email',
     },
+    confirmationEmailUI: {
```

Comment:
The design has the confirmation email field as required.

Reply:
Was technically already required due to not being able to continue without the validation passing, went ahead and added it to the required array anyways.